scripts/convert_stable_diffusion_checkpoint_to_onnx.py

```diff
@@ -69,8 +69,10 @@ def onnx_export(
 
 
 @torch.no_grad()
-def convert_models(model_path: str, output_path: str, opset: int):
-    pipeline = StableDiffusionPipeline.from_pretrained(model_path)
+def convert_models(model_path: str, output_path: str, opset: int, half: bool = False):
+    dtype = torch.float16 if half else torch.float32
+    device = "cuda" if half else "cpu"
```

Comment:
Let's throw a `ValueError` for clarity here if there's no cuda:
```suggestion
    if half and torch.cuda.is_available():
        device = "cuda"
    elif half and not torch.cuda.is_available():
        raise ValueError("`float16` model export is only supported on GPUs with CUDA")
    else:
        device = "cpu"
```